checkov/terraform/graph_builder/variable_rendering/renderer.py

```diff
@@ -25,8 +27,7 @@
 if TYPE_CHECKING:
     from checkov.terraform.graph_builder.local_graph import TerraformLocalGraph
 
-ATTRIBUTES_NO_EVAL = ["template_body", "template"]
-VAR_TYPE_DEFAULT_VALUES = {
+VAR_TYPE_DEFAULT_VALUES: dict[str, list[Any] | dict[str, Any]] = {
```

Comment:
@gruebel Because these changes seem to be type and linting fixes that are not related to my changes, do you mind if I add them in a separate commit to have a clean history? That would mean that the two commits shouldn't be squashed though in this PR.